src/ja/worker/config.py

```diff
@@ -64,6 +64,9 @@ def from_dict(cls, property_dict: Dict[str, object]) -> "WorkerConfig":
             cls._get_dict_from_dict(property_dict=property_dict, key="ssh_config", mandatory=True))
         resource_allocation = ResourceAllocation.from_dict(
             cls._get_dict_from_dict(property_dict=property_dict, key="resource_allocation", mandatory=True))
-        uid = cls._get_str_from_dict(property_dict, "uid", mandatory=False)
+        if "uid" in property_dict:
```

Comment:
Remove this check and instead add "uid" to the list of optional properties in the test that inherits from AbstractSerializableTest.